build.sbt

```diff
@@ -15,9 +15,12 @@ val AkkaBinaryVersion26 = "2.6"
 val AkkaBinaryVersion = if (Nightly) AkkaBinaryVersion26 else AkkaBinaryVersion25
 
 val kafkaVersion = "2.6.0"
+// TODO Jackson is now a provided dependency of kafka-clients
+// https://mvnrepository.com/artifact/org.apache.kafka/kafka-clients/2.6.0
+val jacksonVersion = "2.10.5"
```

Comment:
Could we move the declaration of the `val` elsewhere? (one line up or a few lines down)

It's now sitting between `kafkaVersion` and `embeddedKafkaVersion`/`kafkaVersionForDocs` which will often stay in sync.

Reply:
I'll leave that for the build file rewrite.